packages/banner/src/Banner.story.tsx

```diff
@@ -35,12 +36,20 @@ const meta: StoryMetaType<typeof Banner> = {
     },
   },
   args: {
-    children:
-      'To avoid disrupting majority writes, new members are now added to replica sets as priority=0, votes=0 until they reach secondary state, after which Cloud Manager automatically updates the configuration to match the priority and votes value specified in the deployment.',
+    children: (
+      <>
+        To avoid disrupting majority writes, new members are now added to
+        replica sets as priority=0, votes=0 until they reach secondary state,
+        after which Cloud Manager automatically updates the configuration to
+        match the priority and votes value specified in the deployment. &nbsp;
+        <a href>Anchor tag</a>&nbsp;<Link>Link Component</Link>
```

Comment:
I think you need a valid href, like in callout below, so that the correct cursor shows up